### ISSUE TITLE
Deploy check is failing for user defined networks with static ip assigned to the container

### ISSUE THREAD

**Issue by maruns** [zodern/meteor-up#953]
Deploy check is failing with error
```
              ------------------------------------STDERR------------------------------------
              Too much time spent restarting.
        => Starting meteor app on port:80

              ------------------------------------STDOUT------------------------------------
              Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        Container has no IP Address, likely from it restarting.
        => Container status:
        restarted: 0 times {"Bridge":"","SandboxID":"*","HairpinMode":false,"LinkLocalIPv6Address":"","LinkLocalIPv6PrefixLen":0,"Ports":{"80/tcp":[{"HostIp":"127.0.0.1","HostPort":
"4000"}]},"SandboxKey":"*","SecondaryIPAddresses":null,"SecondaryIPv6Addresses":null,"EndpointID":"","Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"IPAddress":"","IPPrefixLen":0,"IPv6Gateway":"","M
acAddress":"","Networks":{"app0":{"IPAMConfig":{"IPv4Address":"172.19.0.20"},"Links":null,"Aliases":["*"],"NetworkID":"*","EndpointID":"*","Gateway":"172.19.0.1","IPAddress":"172.19.0.20","IPPrefixLen":16,"IPv6Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"MacAddress":"*"}}} {"Status":"running","Running":true,"Paused":f
alse,"Restarting":false,"OOMKilled":false,"Dead":false,"Pid":19395,"ExitCode":0,"Error":"","StartedAt":"2018-06-02T09:28:39.438137575Z","FinishedAt":"0001-01-01T00:00:00Z"}
        => Logs:

        To see more logs type 'mup logs --tail=200'


              ------------------------------------------------------------------------------
```
`*` in logs are places where I've removed values

```
$ docker --version
Docker version 1.13.1, build 94f4240/1.13.1
```

```
$ npm -v mup
5.6.0
```

I've created custom bridged network for my app.
```
docker network create \
    -o "com.docker.network.bridge.name"="app0" \
    -o "com.docker.network.bridge.enable_ip_masquerade"=1 \ 
    -o "com.docker.network.bridge.host_binding_ipv4"="172.19.0.1" \
    --subnet 172.19.0.0/16 app0
```

Extra docker args used in configuration:
```
"--hostname=my.app.com",
"--network=app0",
"--ip=172.19.0.20"
```

Command executed at
https://github.com/zodern/meteor-up/blob/fff181ebe21751ea7687394e12facd368fdf5708/src/plugins/meteor/assets/meteor-deploy-check.sh#L60
will return an empty line in case of this configuration

According to docker docks this line should look like:
```
docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'
```
https://docs.docker.com/engine/reference/commandline/inspect/#get-an-instances-ip-address

**Comment by bgadrian** [comment]
I confirm that, when using custom networks, the Veryfing deployment fails, it uses the internal IP and of course it cannot be accessed.  

It seems that my problem is different, it tries to ping the address `Could not resolve host: 172.17.0.2172.18.0.2`

**Comment by zodern** [comment]
Custom networks is fixed in Mup 1.5.

> It seems that my problem is different, it tries to ping the address Could not resolve host: 172.17.0.2172.18.0.2
This is fixed by #1055

**Comment by Slind14** [comment]
After updating to this version we ran into this: https://github.com/zodern/meteor-up/issues/1110
